### DISCUSSION TITLE
How to create a new record for Catalog Task

### DISCUSSION THREAD

**Discussion by JacobParent** [Snow-Shell/servicenow-powershell#230] [Q&A] [answered]
Hi, I am wondering how I could create a new service now record for a catalog task? 
I currently using the following for creating incidents "New-ServiceNowRecord -Table 'incident' -Values @{}" and I would like to do the same but for a catalog task instead. 
I tried the following "New-ServiceNowRecord -Table 'catalog task' -Values @{}" but it is not working properly... I have the error "201: . " (Do I need to add default values? Maybe I should specify the RITM in advance?)

Thank you for your help!

**Comment by gdbarron** [comment]
What is the full error?  201 is actually a successful http status code.  If you want to associate the task with a request item you should specify `-Values @{'request_item'='ritmxxxxxx'}` or you can use the sys_id of the request item.

**Reply by JacobParent** [reply, marked as answer]
Thank you for your reply :) Finally, I found out it was an access problem (I didn't have the access required).
